spec/lib/phobos_spec.rb

```diff
@@ -11,16 +11,34 @@
   end
 
   describe '.create_kafka_client' do
+    before { Phobos.configure(phobos_config_path) }
+
     it 'returns a new kafka client already configured' do
-      Phobos.configure(phobos_config_path)
+      Phobos.config.logger.ruby_kafka = nil
+      Phobos.configure_logger
 
       expect(Kafka)
         .to receive(:new)
-        .with(hash_including(Phobos.config.kafka.to_hash))
+        .with(hash_including(Phobos.config.kafka.to_hash.merge(logger: nil)))
         .and_return(:kafka_client)
```

Comment:
maybe it doesnt need to return `:kafka_client`, which is a bit of a no op and also make it look like the test below